README.md

```diff
@@ -14,9 +14,9 @@ This theme has been optimized for JavaScript, TypeScript, CSS, SASS(SCSS), HTML
 
 To get the full GitHub experience set the following settings in VSCode:     
 ```
-"editor.fontFamily": "'SF Mono', Menlo, Monaco, 'Courier New', monospace",
+"editor.fontFamily": "'SFMono-Regular','Consolas','Liberation Mono','Menlo','Courier','monospace'",
 "editor.lineHeight": 20,
-"editor.fontSize": 12
+"editor.fontSize": 14
```

Comment:
Pls change back to 12 as discussed

Reply:
Done in my forked copy. Should I make a new PR?

Reply:
It's fine :)